message_ix/model/MESSAGE/model_solve.gms

```diff
@@ -61,15 +61,15 @@ EMISSION_CONSTRAINT.m(node,type_emission,type_tec,type_year)$(
         PRICE_EMISSION.l(node,type_emission,type_tec,year) = - inf ) = 0 ;
 
 * rescale the dual of the emission pool constraint to account for the discount factor and period duration
-EMISSION_POOL_CONSTRAINT.m(node,type_emission,type_tec,year)$(
-    EMISSION_POOL_CONSTRAINT.m(node,type_emission,type_tec,year)) =
-    EMISSION_POOL_CONSTRAINT.m(node,type_emission,type_tec,year) / df_period(year);
+EMISSION_POOL_CONSTRAINT_UP.m(node,type_emission,type_tec,year)$(
+    EMISSION_POOL_CONSTRAINT_UP.m(node,type_emission,type_tec,year)) =
+    EMISSION_POOL_CONSTRAINT_UP.m(node,type_emission,type_tec,year) / df_period(year);
 
 PRICE_EMISSION_POOL.l(node,type_emission,type_tec,year) =
```

Comment:
@behnam-zakeri and @volker-krey shouldn't the PRICE_EMISSION_POOL also be calculated as a result of applying the lower bound not just the upper bound?

Reply:
@Oliver. Thanks for noting this. As discussed personally, we can define a separate price for lower bound too, but it may not be needed at the moment.